app/src/main/web/src/components/GamePage.vue

```diff
@@ -62,7 +62,7 @@ export default {
     correctAnswer: function(){
       return this.questions[this.questionIndex].correct;
     },
-    possibleAnswers: function(){
+    PossibleAnswers: function(){
```

Comment:
can you please check the naming conventions here?

Reply:
Yes, I meant to change it as it was causing some confusion.
I made sure it's consistent else where.